homeassistant/components/balboa/light.py

```diff
@@ -0,0 +1,50 @@
+"""Support for Balboa Spa lights."""
+from __future__ import annotations
+
+from typing import Any, Mapping
+
+from pybalboa import SpaClient, SpaControl
+from pybalboa.enums import OffOnState, UnknownState
+
+from homeassistant.components.light import LightEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import BalboaEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the spa's lights."""
+    spa: SpaClient = hass.data[DOMAIN][entry.entry_id]
+    entities = [BalboaLightEntity(control) for control in spa.lights]
+    async_add_entities(entities)
+
+
+class BalboaLightEntity(BalboaEntity, LightEntity):
+    """Representation of a Balboa Spa light entity."""
+
+    _attr_translation_key = "light"
+
+    def __init__(self, control: SpaControl) -> None:
+        super().__init__(control.client, control.name)
+        self._control = control
+
+    async def async_turn_off(self, **kwargs: Any) -> None:
+        await self._control.set_state(OffOnState.OFF)
+
+    async def async_turn_on(self, **kwargs: Any) -> None:
+        await self._control.set_state(OffOnState.ON)
+
+    @property
+    def is_on(self) -> bool | None:
+        if self._control.state == UnknownState.UNKNOWN:
+            return None
+        return self._control.state != OffOnState.OFF
+
+    @property
+    def translation_placeholders(self) -> Mapping[str, str]:
+        return {"index": f"{self._control.index + 1}" if len(self._client.lights) > 1 else ""}
```

Comment:
If there is only one light, we should not return "", it should be its own translation key

Reply:
Done.